infra/eligibility_screener/template/ecs.tf

```diff
@@ -58,7 +53,7 @@ data "aws_iam_policy_document" "ecr-perms" {
 }
 
 resource "aws_ecr_repository_policy" "eligibility-screener-repo-policy" {
```

Comment:
this seems like it would live in the aws_bootstrap and get created alongside the ECR repo 👋